fastplotlib/graphics/selectors/_rectangle_region.py

```diff
@@ -3,10 +3,10 @@
 
 import pygfx
 
+from ...utils import mesh_masks
 from .._base import Graphic
 from .._features import GraphicFeature
 from ._base_selector import BaseSelector
-from ._mesh_positions import x_right, x_left, y_top, y_bottom
```

Comment:
This import was impossible, so I guess this module is not imported by default?

Reply:
Yes, most of the code in selectors is a playground for now. Only linear and linear region are functional and publicly exposed right now.